src/back/project/src/model/di_mode.cpp

```diff
@@ -0,0 +1,50 @@
+#include "di_mode.hpp"
+#include <shared/errors.hpp>
+
+namespace svetit::project {
+
+/*static*/ std::string DiMode::ToString(const DiMode::Type& diMode)
+{
+	switch (diMode) {
+	case ReadonlyFlag:
+		return "readonly_flag";
+	case ReadwriteFlag:
+		return "readwrite_flag";
```

Comment:
Давай пожалуйста enum ы тут и в базе сразу переведём в camel
```suggestion
		return "readonlyFlag";
	case ReadWriteFlag:
		return "readWriteFlag";
```

Reply:
Готово.